modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -358,6 +358,8 @@ class ApiClient:
         """
         if obj is None:
             return None
+        elif isinstance(obj, Enum):
+            return obj.value
```

Comment:
would it be possible to write a unit test for sanitize_for_serialization with enum ?

Reply:
Done